python/lsst/pipe/tasks/postprocess.py

```diff
@@ -1081,12 +1080,12 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
         butlerQC.put(expCatalog, outputRefs.visitSummary)
 
     def _combineExposureMetadata(self, visit, dataRefs, isGen3=True):
-        """Make a combined exposure catalog from a list of dataRefs.
+        """Make a combined exposure catalog from a list of calexp dataRefs.
```

Comment:
I think I remember something about trying to avoid using the word "calexp" in user-facing places like docstrings, or did I make that up?